README.md

```diff
@@ -1,3 +1,5 @@
+<img align="right" width="200px" src="https://github.com/FluxML/Optimisers.jl/raw/master/docs/src/assets/logo.png">
```

Comment:
```suggestion
<img align="right" width="200px" src="https://fluxml.ai/assets/logo.png">
```

Reply:
This is the big Flux logo. My thinking was just to add the small one, as on most of the other repositories.

Reply:
https://avatars.githubusercontent.com/u/26222520 might be a better permalink then. I'd also be fine with dumping the small version into https://github.com/FluxML/fluxml.github.io/tree/main/_assets so we have a central place for it across the org.

Reply:
Yes that's probably ideal.